### PR TITLE
971 docs db types

### PR DESCRIPTION
**Pull request by npetzall**
_D__repos_npetzall_schemaspy_PR_docs_build_html_configuration_databaseType html

### CODE REVIEW

File: docs/source/_ext/schemaspy_database_types.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                        lines_no_comments = [x for x in stream.readlines() if x[0:1] != "#"]
+                    lines_merged = []
+                    append = False
+                    for line in lines_no_comments:
+                        if append:
+                            lines_merged[len(lines_merged) - 1] += line
+                        else:
+                            lines_merged.append(line)
+                        append = line[-1] == "\\"
+                    db_type = {x[0].strip(): x[1].strip() for x in [y.split("=", 1) for y in lines_merged] if len(x) == 2}
```
Comment (jesperolsson-se) [inline docs/source/_ext/schemaspy_database_types.py:28]: Do less stuff per line of code.

File: docs/source/_ext/schemaspy_database_types.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        rst_text = ""
+        rst_text += ".. flat-table:: \n"
+        rst_text += "   :class: clean-table\n"
+        rst_text += "   :header-rows: 1\n\n"
+        rst_text += "   * - Dbms\n"
+        rst_text += "     - Description\n"
+        rst_text += "     - Argument `-t`\n"
+        for key in sorted(per_dbms):
+            rst_text += "   * - :rspan:`" + str(len(per_dbms[key])-1) + "` "+ key +"\n"
+            for i, v in enumerate(per_dbms[key]):
```
Comment (jesperolsson-se) [inline docs/source/_ext/schemaspy_database_types.py:40]: Rename `i` and `v`

**Review by jesperolsson-se — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -218,7 +218,8 @@ __Linux__ `http://localhost:9000`
 __Windows__ `http://$(boot2docker ip):9000`  
 
 ## Documentation
-You'll need sphinx installed http://www.sphinx-doc.org    
+You'll need sphinx installed http://www.sphinx-doc.org  
+Install linuxdoc, sphinx_rtd_theme, `pip install linuxdoc sphinx_rtd_theme`
```
[the remark this answers is not in the document]
Comment (npetzall) [reply README.md:222]: I'm not actually sure what you mean.
Add `requirements.txt` or just say that they are dependencies and let the user figure out how to install them.

[the remark this answers is not in the document]
Comment (jesperolsson-se) [reply README.md:222]: I just meant: Don't mention `linuxdoc` and `sphinx_rtd_theme` twice (text + command).
I'd be satisfied with, for example, the following:
> You'll need sphinx installed http://www.sphinx-doc.org  
> with these dependencies `pip install linuxdoc sphinx_rtd_theme`  

A `requirements.txt` would be a good approach. I'll leave it for you to decide if it should be part of the scope of this ticket/PR.

[the remark this answers is not in the document]
Comment (npetzall) [reply README.md:222]: I don't do python that often(currently I change computers more often), and there is more than one way to set it up. So I'm currently leaning towards building a container for building the docs. But that is a separate ticket.

**Review by jesperolsson-se — APPROVED** [review]

**Pull request merged**